src/components/common/playlist/PlaylistDetails.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Image, PixelRatio, StyleSheet, Text, View } from "react-native";
+import PropTypes from "prop-types";
+import MoreIcon from "../icons/MoreIcon";
+import PlayIcon from "../icons/PlayIcon";
+import { handleStartPlaylist } from "../../../utils/MusicPlayerUtil";
+import { COLOR_PALETTE } from "../../../utils/constants";
+
+const PlaylistDetails = ({ playlist }) => {
+	const onPress = async () => {
+		await handleStartPlaylist(playlist?.songs);
+	};
+	return (
+		<>
+			<View style={styles.artworkContainer}>
+				<Image
+					style={styles.tinyLogo}
+					source={{
+						uri: playlist?.artwork
+					}}
+				/>
+			</View>
+			<Text style={styles.playlistName}>{playlist?.name}</Text>
+			<Text style={styles.funText}>Songs: {playlist?.songs?.length}</Text>
+			<View style={styles.playlistControls}>
+				<MoreIcon />
+				<PlayIcon onPress={onPress} />
+			</View>
+		</>
+	);
+};
+
+PlaylistDetails.propTypes = {
+	playlist: PropTypes.object.isRequired
+};
+
+export default PlaylistDetails;
+
+const styles = StyleSheet.create({
+	artworkContainer: {
+		flex: 1,
+		justifyContent: "center",
+		alignItems: "center",
+		marginBottom: 16
+	},
+	tinyLogo: {
+		width: 175,
```

Comment:
can you use, Dimensions instead? that way it's always proportional to screen size